collin/index.html

```diff
@@ -0,0 +1,31 @@
+<!-- <!DOCTYPE html> -->
```

Comment:
Since we’re not working in StackBlitz anymore, let’s go ahead and I comment this out so we have fully valid HTML! This is the way